### ISSUE TITLE
Env variables are attached to build but they must not

### ISSUE THREAD

**Issue by timbset** [reimagined/resolve#616]
I use following part of `resolve.config.json`:
```
"storage": {
  "adapter": "${RESOLVE_STORAGE_ADAPTER}",
  "options": {
    "url": "mongodb://localhost:27017/Database",
    "collection": "Events"
  }
}
```
1. I'm trying to build an app without specified variable - I see an error:
```
ERROR in ./node_modules/resolve-scripts/dist/runtime/server/event_store.js
Module not found: Error: Can't resolve 'D:\Repositories\okrserver' in 'D:\Repositories\okrserver\node_modules\resolve-scripts\dist\runtime\server'
 @ ./node_modules/resolve-scripts/dist/runtime/server/event_store.js 21:21-54
 @ ./node_modules/resolve-scripts/dist/runtime/server/socket_handler.js
 @ ./node_modules/resolve-scripts/dist/runtime/server/index.js
 @ multi babel-regenerator-runtime ./node_modules/resolve-scripts/dist/runtime/server/index.js
```
This means that builder tries to include chosen storage to the bundle. But I should not choose it on build. I should choose it on an app start.

2. I built an app with that variable and I try to start an app with *other* variable value. My app is started with settings from build. So there is no way to customize settings on an app start, only in its bundle.

**Comment by TatyanaRyzh** [comment]
Hi timbset,
 
We've reproduced your scenario and will fix it as soon as possible.
 
Thank you for helping us improve reSolve!

**Comment by TatyanaRyzh** [comment]
Fixed in [V0.12.1](https://github.com/reimagined/resolve/releases/tag/V0.12.1)
